<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1601</width>
    <height>890</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>0</y>
      <width>1601</width>
      <height>921</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="Listespec">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-60</y>
        <width>281</width>
        <height>971</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="recc.qrc">:/resourcefile/Rectangle 2.png</pixmap>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>40</y>
        <width>271</width>
        <height>151</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="recc.qrc">:/resourcefile/Tv_tally_white 1.png</pixmap>
      </property>
     </widget>
     <widget class="QTableView" name="tableView_2">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>250</y>
        <width>1271</width>
        <height>601</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/resourcefile/Spectateurs.png);
QTableView
{
	background-color: rgb(255, 255, 255);
	border: none;
	color: rgb(0, 0, 0);
}
QTableView::item:selected
{
	color:white;
	background-color: rgb(93, 93, 93);
	border: none;
}

QHeaderView::section
{
	border: none;
	background-color: qlineargradient(spread:pad, x1:0.989474, y1:0.977, x2:0.0214737, y2:0.085, stop:0 rgba(0, 0, 0, 0), stop:1 rgba(255, 255, 255, 0));
	color: rgb(172, 172, 172);
}</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="LineChercher_2">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>180</y>
        <width>741</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="BoutonTrier">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>80</y>
        <width>41</width>
        <height>29</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/resourcefile/numerical-sorting-48.png);
border:none;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_PDF">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>80</y>
        <width>83</width>
        <height>29</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/resourcefile/pdf-48.png);
border:none;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_statistique">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>80</y>
        <width>61</width>
        <height>29</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/resourcefile/statistics-48.png);
border:none;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_gotomodifier">
      <property name="geometry">
       <rect>
        <x>1430</x>
        <y>210</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/resourcefile/pen 1.png);
border: none;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>60</width>
        <height>60</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="BoutonSupprimer_2">
      <property name="geometry">
       <rect>
        <x>1480</x>
        <y>200</y>
        <width>61</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/resourcefile/trash 1.png);
border: none;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="Ajouter_2">
      <property name="geometry">
       <rect>
        <x>1310</x>
        <y>90</y>
        <width>231</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/resourcefile/ajoutspec.png);
border: none;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>140</y>
        <width>221</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Chercher un spectateur :</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Boutonchercher">
      <property name="geometry">
       <rect>
        <x>1020</x>
        <y>190</y>
        <width>83</width>
        <height>29</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/resourcefile/search-32.png);
border:none;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="ajouterspec">
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1651</width>
        <height>991</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/resourcefile/ajout.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>220</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>ID</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>360</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Prénom</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>440</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Adresse mail</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>520</y>
        <width>161</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Numéro de téléphone</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>600</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Adresse</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>680</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Age</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_id_2">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>250</y>
        <width>341</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_prenom_2">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>390</y>
        <width>341</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_mail_2">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>470</y>
        <width>341</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_num_2">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>550</y>
        <width>341</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_adresse_2">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>630</y>
        <width>341</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="age">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>710</y>
        <width>341</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_ajouter_2">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>750</y>
        <width>331</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/resourcefile/Btnajout.png);
border:none;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_nom_2">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>320</y>
        <width>341</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>290</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Nom</string>
      </property>
     </widget>
     <widget class="QPushButton" name="BoutonModifier_2">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>800</y>
        <width>331</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/resourcefile/Btnmodifier.png);
border:none;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="retour">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>51</width>
        <height>29</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/resourcefile/arrow-97-48.png);
border:none;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="recc.qrc"/>
 </resources>
 <connections/>
</ui>
